src/UnisonShare/Page/ProjectPage.elm

```diff
@@ -908,8 +908,7 @@ viewUseProjectModal project branchRef =
             ProjectRef.toString project.ref
 
         libVersion v =
-            "lib."
-                ++ ProjectSlug.toNamespaceString (ProjectRef.slug project.ref)
+            ProjectSlug.toNamespaceString (ProjectRef.slug project.ref)
```

Comment:
I think this is needed in some of the places where `libVersion` is used—like in the `pullCommand`